random-beans/src/main/java/io/github/benas/randombeans/randomizers/registry/TimeRandomizerRegistry.java

```diff
@@ -66,7 +67,8 @@ public void init(EnhancedRandomParameters parameters) {
         randomizers.put(LocalDateTime.class, new LocalDateTimeRangeRandomizer(of(minDate, minTime), of(maxDate, maxTime), seed));
         randomizers.put(LocalTime.class, new LocalTimeRangeRandomizer(minTime, maxTime, seed));
         randomizers.put(MonthDay.class, new MonthDayRandomizer(seed));
-        randomizers.put(OffsetDateTime.class, new OffsetDateTimeRandomizer(seed));
+        randomizers.put(OffsetDateTime.class,
```

Comment:
Not sure this is the correct thing to do (similar to https://github.com/benas/random-beans/pull/333).